src/operations/booleans/fromFakePolygons.js

```diff
@@ -0,0 +1,42 @@
+const {vec2} = require('../../math')
+
+const {geom2} = require('../../geometry')
+
+const fromFakePolygon = (polygon) => {
+  // this can happen based on union, seems to be residuals -
+  // return null and handle in caller
+  if (polygon.vertices.length < 4) {
+    return null
```

Comment:
should be undefined rather than null ? (that is the case in most other parts of the updated code, although I recall a possible error in null vs undefined somewhere ?)

Reply:
I'm not sure so let's leave this for now.

Reply:
I this because this is in part legacy code ? In general null should be avoided for coherence imho